voxblox_ros/src/tsdf_server.cc

```diff
@@ -313,11 +310,10 @@ bool TsdfServer::generateMesh() {
   generate_mesh_timer.Stop();
 
   timing::Timer publish_mesh_timer("mesh/publish");
-  visualization_msgs::MarkerArray marker_array;
-  marker_array.markers.resize(1);
-  fillMarkerWithMesh(mesh_layer_, color_mode_, &marker_array.markers[0]);
-  marker_array.markers[0].header.frame_id = world_frame_;
-  mesh_pub_.publish(marker_array);
+  voxblox_msgs::Mesh mesh_msg;
+  fillVoxbloxMeshWithMesh(mesh_layer_, color_mode_, &mesh_msg);
```

Comment:
nit: name is a bit strange, maybe convert/generateToVoxbloxMeshMsg?

Reply:
The other functions that do similar things are all called fill____WithMesh, though I agree it sounds weird. Will rename